core/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.library'
+apply from: '../dependencies.gradle'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "${KIN_ECOSYSTEM_CORE_VERSION_NAME}"
+    }
+}
+
+dependencies {
+
+    implementation project(':common')
+
+    api devDependencies.kinCore
```

Comment:
why api is kinCore api used outside of com.kin.ecosystem.core?

Reply:
yes on the SDK, we init the blockchainsource with kinClient